ispyb/sp/xtalimaging.py

```diff
@@ -49,6 +49,50 @@ def upsert_sample_image_auto_score(
             args=[image_full_path, schema_name, score_class, probability],
         )
 
+    def insert_subsample_for_image_full_path(
+        self,
+        image_full_path,
+        source,
+        position1x,
+        position1y,
+        position2x=None,
+        position2y=None,
+    ):
+        """Store new subsample for a given sample image. Either specify a point
+        (by providing position1x and position1y) or a ROI box (by additionally
+        providing position2x and position2y). Position coordinates are given in
+        pixels from the top-left corner of the image.
```

Comment:
```suggestion
        """Store new subsample for a given sample image.
        
        Either specify a point (by providing position1x and position1y)
        or a ROI box (by additionally providing position2x and position2y).
        Position coordinates are given in pixels from the top-left corner
        of the image.
```

>Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings